apps/prairielearn/src/tests/chunks.test.js

```diff
@@ -43,7 +43,7 @@ const COURSE = {
 };
 
 function getAllChunksForCourse(course_id) {
```

Comment:
Should this function be async? It's returning a promise.

Reply:
Changed in 1e05e18.